utility/src/main/java/org/oppia/util/logging/EventBundleCreator.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.util.logging
+
+import android.os.Bundle
+import org.oppia.app.model.EventLog
+
+/**
+ * Utility for creating bundles from [EventLog] objects.
+ * Note that this utility may later upload them to remote services.
+ */
+interface EventBundleCreator {
```

Comment:
You can directly make this a utility class instead of creating an interface and having others implement it . Both your fake and your real event loggers can depend on the utility class

Reply:
Yep, done !